testing/tests/DevExpress.ui.widgets/button.tests.js

```diff
@@ -525,3 +544,39 @@ QUnit.module('templates', () => {
         assert.strictEqual(checkStyleHelper.getWhiteSpace($template[0].parentNode), 'normal', 'whiteSpace');
     });
 });
+
+QUnit.module('events subscriptions', () => {
+    QUnit.test('click', function(assert) {
+        const clickHandler = sinon.spy();
+        const $button = $('#button').dxButton({
+            text: 'test'
+        });
+        const button = $button.dxButton('instance');
+
+        button.on('click', clickHandler);
+
+        $button.trigger('dxclick');
+
+        assert.ok(clickHandler.calledOnce, 'Handler should be called');
+        const params = clickHandler.getCall(0).args[0];
+        assert.ok(params, 'Event params should be passed');
+        assert.ok(params.event, 'Event should be passed');
+        assert.ok(params.validationGroup, 'validationGroup should be passed');
+    });
+
+    QUnit.test('contentReady', function(assert) {
+        assert.expect(3);
+
+        const eventHandler = (e) => {
```

Comment:
sinon.spy?

Reply:
We can use sinon.spy here but i do not see any profits in this case

Reply:
There is no need to extract handler. We can remove `const` and define handler as inline function.

```js
button.on('contentReady', (e) => {
 //handler
});
```